frontend/css/brainzplayer.less

```
@@ -351,6 +351,23 @@
   }
 }
 
+.volume {
+  position: fixed;
+  width: fit-content;
+  z-index: 109;
+  height: 10em;
+  background-color: #f8f8f8;
+  right: 16em;
+  .volume-input {
+    width: 7em;
```

I think this is a bit wide, I suggest:
```suggestion
    width: 3.5em;
```